docs/sources/alerting/notifications.md

```diff
@@ -129,6 +129,17 @@ Move any existing rules using `custom_details.myMetric` to `custom_details.queri
 This behavior will become the default in a future version of Grafana.
 
 > Using `dedup_key` tag will override Grafana generated `dedup_key` with a custom key.
+
+### VictorOps
+
+To set up VictorOps, provide the URL from the Grafana Integration and substitute `$routing_key` with the routing key.
+
+Setting | Description
+---------- | -----------
+Auto resolve incidents | Resolve incidents in VictorOps once the alert goes back to ok
+
+>**Note:** The tag `Severity` has special meaning in the [VictorOps Incident Fields](https://help.victorops.com/knowledge-base/incident-fields-glossary/). If an alert panel defines this tag key, then it is transposed to the root of the event sent to VictorOps.
```

Comment:
Is "transposed" the right word? If something is transposed, then it switches places with something else.

Perhaps you mean appended, which is adding it to the end?

Reply:
copied that language from [here](https://github.com/grafana/grafana/blame/460883d0b291a5e497a07e6efd7c8ef88a507fa0/docs/sources/alerting/notifications.md#L125)

It is transposed; it replaces a field in the root of the event. Let me know if I should change this wording and the other please!

Reply:
In that case, It needs to be something like

transposed with the root

or

it replaces the root 

I recommend the second option, as simpler words are easier for not-native English speakers to understand.

Reply:
```suggestion
>**Note:** The tag `Severity` has special meaning in the [VictorOps Incident Fields](https://help.victorops.com/knowledge-base/incident-fields-glossary/). If an alert panel defines this tag key, then it replaces the `message_type` in the root of the event sent to VictorOps.
```